omero/downloads/cli/help.out

```diff
@@ -1,49 +1,141 @@
+  --logs                                Upload log file (if any) with report. Required --report (**)
+  --email EMAIL                         Email for reported errors. Required --report (**)
+
+Annotation arguments:
+  Optional arguments passed strictly to Java allowing to annotate imports.
+
+  --annotation-ns ANNOTATION_NS         Namespace to use for subsequent annotation (**)
+  --annotation-text ANNOTATION_TEXT     Content for a text annotation (**)
+  --annotation-link ANNOTATION_LINK     Comment annotation ID to link all images to (**)
+
+Java arguments:
+  Optional arguments passed strictly to Java.
+
+  -f                                    Display the used files and exit (**)
+  -c                                    Continue importing after errors (**)
+  -l READER_FILE, --readers READER_FILE
+                                        Use the list of readers rather than the default (**)
+  -d DATASET_ID                         OMERO dataset ID to import image into (**)
+  -r SCREEN_ID                          OMERO screen ID to import plate into (**)
+  -T TARGET, --target TARGET            OMERO target specification (**)
+  --debug LEVEL                         Turn debug logging on (**)
+  --output TYPE                         Set an alternative output style
+  --encrypted TYPE                      Whether the import should use SSL or not
+
+Advanced Java arguments:
+  Optional arguments passed strictly to Java. For more information, see --advanced-help.
+
+  --advanced-help                       Show the advanced help text
+  --transfer [TYPE]                     Transfer methods like in-place import
+  --exclude [TYPE]                      Exclusion filters for preventing re-import
+  --checksum-algorithm [TYPE]           Alternative hashing mechanisms balancing speed & accuracy
+  --parallel-upload COUNT               Number of file upload threads to run at the same time
+  --parallel-fileset COUNT              Number of fileset candidates to import at the same time
 
 Examples:
 
-  $ importer-cli -s localhost -u user -w password -d 50 foo.tiff
-  $ importer-cli -s localhost -u user -w password -d Dataset:50 foo.tiff
-  $ importer-cli -f foo.tiff
-  $ importer-cli -s localhost -u username -w password -d 50 --debug ALL foo.tiff
+  # Display help
+  $ omero import -h
+  # Import foo.tiff using current login
+  $ omero import ~/Data/my_file.dv
+  # Import foo.tiff using input credentials
+  $ omero import -s localhost -u user -w password foo.tiff
+  # Set Java debugging level to ALL
+  $ omero import foo.tiff -- --debug=ALL
+  # Display used files for importing foo.tiff
+  $ omero import foo.tiff -f
+  # Limit debugging output
+  $ omero import -- --debug=ERROR foo.tiff
 
 For additional information, see:
 https://docs.openmicroscopy.org/latest/omero/users/cli/import.html
-Report bugs at https://www.openmicroscopy.org/forums
+Report bugs to <ome-users@lists.openmicroscopy.org.uk>
```

Comment:
I don't think that what should be recommended

Reply:
That's what `--help` prints out.

Reply:
In that case we should update the output of the command

Reply:
see https://github.com/ome/omero-py/pull/297